ios-device-lib-stdio-handler.js

```diff
@@ -11,16 +11,23 @@ const Constants = require("./constants");
 const HeaderSize = 4;
 
 class IOSDeviceLibStdioHandler extends EventEmitter {
-	constructor() {
+	constructor(showDebugInformation) {
```

Comment:
It's better to accept the whole `options` object here and set `this._showDebugInformation = options.debug`.
That way it'll be easier to extend and it'll be clearer IMO because you'd pass `{ debug: true }` instead of just a boolean